### ISSUE TITLE
Keep background after update

### ISSUE THREAD

**Issue by L4ngeFrank** [basnijholt/lovelace-ios-dark-mode-theme#51]
Hey Bas, great theme, love it!!!  BUT, after every update the background (my favorite: https://github.com/basnijholt/lovelace-ios-dark-mode-theme/blob/master/backgrounds/homekit-bg-4.jpeg)  restores to the original green one. Is there a solution to keep that setting saved after every update? 

Gr uit Brabant ;)

**Comment by basnijholt** [comment]
Yes! The solution is to install https://github.com/basnijholt/lovelace-ios-themes

Which is better in many ways :)

**Comment by L4ngeFrank** [comment]
THANKS!! Afterall you made it us so easy. One more question: is it possible to change the color of the switches?
Like the green color into orange or the button in another color?
![switch](https://user-images.githubusercontent.com/37044307/81476181-b90d2580-9210-11ea-8908-36e0e26b9f6b.png)
